pallets/parachain-staking/src/rewards/minting_rewards.rs

```diff
@@ -0,0 +1,88 @@
+use crate::{traits::PayoutReward, BalanceOf, Config, Event, Pallet, RoundIndex, WeightInfo};
+use frame_support::{
+	pallet_prelude::Weight,
+	traits::{tokens::currency::Currency, Imbalance},
+};
+use sp_runtime::DispatchError;
+
+pub struct MintingRewards;
```

Comment:
```suggestion
pub struct MintRewards;
```
also would suggest renaming this to be consistent with `TransferFromRewardsAccount`: